site/setup.cfg

```diff
@@ -10,6 +10,7 @@ python_requires = >=3.9
 zip_safe = False
 install_requires =
     python-ldap>=3.4.0,<3.5.0
+    #missing invenio-rdm-migrator?
```

Comment:
it is missing because of the sqlalchemy conflict but you can add it back now when your fix is merged

Reply:
sorry, missed this commented our piece
invenio-rdm-migrator will not be installed here at all